src/cco-imports/bfo-core.ttl

```diff
@@ -380,7 +380,7 @@ skos:scopeNote rdf:type owl:AnnotationProperty .
                                              rdfs:range <http://purl.obolibrary.org/obo/BFO_0000003> ;
                                              dc11:identifier "078-BFO" ;
                                              rdfs:label "temporal part of"@en ;
-                                             skos:definition "b temporal part of c =Def b occurrent part of c & (b and c are temporal regions) or (b and c are spatiotemporal regions & b temporally projects onto an occurrent part of the temporal region that c temporally projects onto) or (b and c are processes or process boundaries & b occupies a temporal region that is an occurrent part of the temporal region that c occupies)"@en ;
+                                             skos:definition "b temporal_part_of c iff b and c are both instances of Occurrent and if b and c are both instances of Temporal Region then b_occurrent_part_of c or if b and c are instances of Spatiotemporal Region & there exists an instance of Occurrent z and b temporally_projects_onto z and there exists an instance of Temporal Region w and z is occurrent_part_of w then c temporally_projects_onto w or if b and c are instances of Process or Process Boundary & there exists an instance of Temporal Region z and z is_temporal_region_of b and there exists an instance of Temporal Region w and z occurrent_part_of w then w is_temporal_region of c."@en ;
```

Comment:
Yikes. This make an already hard definition to read harder. Too long to no benefit I see